swim/node.go

```diff
@@ -510,3 +510,7 @@ func (n *Node) pingNextMember() {
 		"numErrors": len(errs),
 	}).Warn("ping request inconclusive due to errors")
 }
+
```

Comment:
Your point about future compatibility makes a lot of sense, how about `GetReachableMembers`. Using `Pingable` feels wrong if you look at the definition of `[Memberlist::Pingable(Member)](https://github.com/uber/ringpop-go/blob/feature/get-members/swim/memberlist.go#L127-L131)` where a pingable member could never be its own host, while its own host is definitely part of the list of all 'reachable' nodes.